### ISSUE TITLE
Language server dies due to missing dependency `globrex`

### ISSUE THREAD

**Issue by brunnerh** [sveltejs/language-tools#2575]
### Describe the bug

Probably due to 7c23767.

`globrex` was added in `devDependencies`, maybe it should be a regular dependency?

```
Error: Cannot find module 'globrex'
Require stack:
- c:\Users\<user>\.vscode\extensions\svelte.svelte-vscode-109.2.0\node_modules\svelte-language-server\dist\src\plugins\css\global-vars.js
- c:\Users\<user>\.vscode\extensions\svelte.svelte-vscode-109.2.0\node_modules\svelte-language-server\dist\src\plugins\css\CSSPlugin.js
- c:\Users\<user>\.vscode\extensions\svelte.svelte-vscode-109.2.0\node_modules\svelte-language-server\dist\src\plugins\index.js
- c:\Users\<user>\.vscode\extensions\svelte.svelte-vscode-109.2.0\node_modules\svelte-language-server\dist\src\server.js
- c:\Users\<user>\.vscode\extensions\svelte.svelte-vscode-109.2.0\node_modules\svelte-language-server\bin\server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1232:15)
    at Module._load (node:internal/modules/cjs/loader:1058:27)
    at c._load (node:electron/js2c/node_init:2:16955)
    at Module.require (node:internal/modules/cjs/loader:1318:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (c:\Users\<user>\.vscode\extensions\svelte.svelte-vscode-109.2.0\node_modules\svelte-language-server\dist\src\plugins\css\global-vars.js:9:35)
    at Module._compile (node:internal/modules/cjs/loader:1484:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1564:10)
    at Module.load (node:internal/modules/cjs/loader:1295:32)
    at Module._load (node:internal/modules/cjs/loader:1111:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'c:\\Users\\<user>\\.vscode\\extensions\\svelte.svelte-vscode-109.2.0\\node_modules\\svelte-language-server\\dist\\src\\plugins\\css\\global-vars.js',
    'c:\\Users\\<user>\\.vscode\\extensions\\svelte.svelte-vscode-109.2.0\\node_modules\\svelte-language-server\\dist\\src\\plugins\\css\\CSSPlugin.js',
    'c:\\Users\\<user>\\.vscode\\extensions\\svelte.svelte-vscode-109.2.0\\node_modules\\svelte-language-server\\dist\\src\\plugins\\index.js',
    'c:\\Users\\<user>\\.vscode\\extensions\\svelte.svelte-vscode-109.2.0\\node_modules\\svelte-language-server\\dist\\src\\server.js',
    'c:\\Users\\<user>\\.vscode\\extensions\\svelte.svelte-vscode-109.2.0\\node_modules\\svelte-language-server\\bin\\server.js'
  ]
}
```

### Reproduction

Install VS Code extension @ `v109.2.0`

### Expected behaviour

It does not die.

### System Info

- OS: Win 11
- IDE: VSCode

**Comment by bozgi** [comment]
Can confirm that this is caused by new version, you can make it work by rollbacking to v109.1.0.

**Comment by SousaFelipe** [comment]
I had the same problem on Windows 11 and fixed it by going to the extension installation folder "C:\Users\\**MY_USER_NAME**\\.vscode\extensions\svelte.svelte-vscode-109.2.0\" and running the command `npm install globrex`. Well, it might not be the most elegant way to fix this, but at least it works again.

**Comment by jasonlyu123** [comment]
Just released 109.2.1. It should be fixed now.
